tensorflow_recommenders_addons/dynamic_embedding/python/ops/dynamic_embedding_variable.py

```diff
@@ -305,6 +306,13 @@ def _get_default_devices():
           [dtypes.int64, dtypes.int64],
           [dtypes.int32, dtypes.float32],
       ]
+    if is_macos() and is_arm64():
+      if key_dtype == dtypes.half or value_dtype == dtypes.half:
+        raise TypeError("""
+          Key or value dtype contains float16, 
```

Comment:
'Key or' is no needed.

Reply:
"""
          `float16` value dtype is not supported on macOS with ARM64 architecture. Please try another type.
         """